src/bctklib/smart-contract/TestApplicationEngine.cs

```diff
@@ -309,17 +296,5 @@ private void OnNotify(object? sender, NotifyEventArgs args)
         }
 
         bool CheckWitnessOverride(byte[] hashOrPubkey) => witnessChecker(hashOrPubkey);
-
-        protected override void OnSysCall(uint methodHash)
```

Comment:
Any reason why you remove this?

Reply:
The ApplicationEngine class removes the OnSysCall method.

Reply:
You use `JumpTable`